frame/grandpa/src/lib.rs

```diff
@@ -244,10 +244,11 @@ pub mod pallet {
 			origin: OriginFor<T>,
 			delay: T::BlockNumber,
 			best_finalized_block_number: T::BlockNumber,
-		) -> DispatchResultWithPostInfo {
+		) -> DispatchResult {
```

Comment:
Why does this change?

Reply:
Bc. the function returned `()`, there was no PostInfo attached. So a simple DispatchResult is sufficient.